sys/include/net/netopt.h

```diff
@@ -199,6 +199,14 @@ typedef enum {
      */
     NETOPT_CCA_THRESHOLD,
 
+    /**
+     * @brief get statistics about sent and received packets of the layer
```

Comment:
```of the layer``` doesn't seem clear to me, but ```device``` is probably not accurate if we use e.g., a MAC layer on top of a radio device, right?